src/pages/workspace/WorkspacesListPage.tsx

```diff
@@ -119,6 +119,12 @@ function WorkspacesListPage() {
     const [policyNameToDelete, setPolicyNameToDelete] = useState<string>();
     const isLessThanMediumScreen = isMediumScreenWidth || shouldUseNarrowLayout;
 
+    // We need this to update translation for deleting a workspace when it has third party card feeds or expensify card assigned.
+    const workspaceAccountID = PolicyUtils.getWorkspaceAccountID(policyIDToDelete ?? '-1');
+    const [cardFeeds] = useOnyx(`${ONYXKEYS.COLLECTION.SHARED_NVP_PRIVATE_DOMAIN_MEMBER}${workspaceAccountID}`);
+    const [cardsList] = useOnyx(`${ONYXKEYS.COLLECTION.WORKSPACE_CARDS_LIST}${workspaceAccountID}_${CONST.EXPENSIFY_CARD.BANK}`);
+    const hasCardFeedOrExpensifyCard = !isEmptyObject(cardFeeds) || !isEmptyObject(cardsList);
+
```

Comment:
Instead of reusing the same code as in `WorkspaceProfilePage.tsx`, could you create an util function instead to respect the `DRY` principle?

Reply:
sure, will update soon

Reply:
Im not sure where I need to move and will it make sense to be honest as I still need to get cardFeeds, cardsList and workspaceAccountID in each component and the only thing will be to calculate **hasCardFeedOrExpensifyCard** and taking into account that isEmptyObject is imported in each page Im not sure I need to move it somewhere

Reply:
Sorry @ishpaul777, it got lost in my GH notifications. 

Alright, fine. Let's not abstract it or create a hook, as it'd be overkill given the complexity and current scope .